### PR TITLE
make datapipes.healpix importable 

### PR DESCRIPTION
**Pull request by yairchn**
## Description
in the current code `from modulus.datapipes.healpix import CoupledTimeSeriesDataModule` does not work. This PR closes issue #680 

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [x] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [x] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

**Review by akshaysubr — APPROVED** [review]

**Review by daviddpruitt — APPROVED** [review]

File: modulus/datapipes/__init__.py [hunk 1]
```diff
@@ -13,3 +13,5 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from . import healpix
```
Comment (dallasfoster) [inline modulus/datapipes/__init__.py:17]: Hi @yairchn, to follow convention for the rest of the datapipes, I am inclined to suggest removing this import statement. The reason is that each of the submodule datapipes may require various non-required packages and putting them in this level will require all `modulus.datapipe` users to have those dependencies. In this case, the use of `omegaconf` is an optional dependency in modulus but required for `healpix` datapipes.

**Review by dallasfoster — CHANGES_REQUESTED** [review]

**Review by dallasfoster — APPROVED** [review]

**Pull request merged**